hudi-spark-datasource/hudi-spark3.3.x/src/main/scala/org/apache/spark/sql/parser/HoodieSpark3_3ExtendedSqlParser.scala

```diff
@@ -121,11 +120,7 @@ class HoodieSpark3_3ExtendedSqlParser(session: SparkSession, delegate: ParserInt
 
   private def isHoodieCommand(sqlText: String): Boolean = {
     val normalized = sqlText.toLowerCase(Locale.ROOT).trim().replaceAll("\\s+", " ")
-    normalized.contains("system_time as of") ||
```

Comment:
we are ok getting rid of all of these aliases?

Reply:
Sorry, I didn’t quite understand what you meant. Could you explain in more detail?

Reply:
I think Jon is talking about the compatibility issue too.

Reply:
I think Jon is talking about the compatibility issue too.

Reply:
The purpose of this MR is to align with Spark's SQL semantics. Given Spark's parsing approach, it seems that directly using Spark's Analyzer for compatibility transformation is not yet feasible. My suggestion is either to temporarily hold off on introducing this MR or to directly ensure full consistency with Spark's semantics and expression parsing.

Reply:
The compatibility issue is critical, otherwise the old sql job can not migrate.

Reply:
got it, close this MR

Reply:
spark will analyze TimesStamp on ResolveRelation, can not inject our analyze rule in advance, close it before find other ways.